doc/introduction/new_opmath.rst

```diff
@@ -0,0 +1,436 @@
+.. _new_opmath:
+
+Updated Operators
+=================
+
+In version ``0.36`` of PennyLane we changed some things behind the scenes on how operators and arithmetic operations between them are handled.
+This realizes a few objectives:
+
+1. To make it as easy to work with PennyLane operators as it would be with pen and paper.
+2. To improve the efficiency of operator arithmetic.
+
+In many cases, these changes should not break code and the difference to previous
+versions may not be noticeable.
+
+This page provides additional details about operator arithmetic updates and can be
+used to troubleshoot issues for those affected users.
+
+.. note::
+
+    If you are looking for a quick fix, jump to the :ref:`Troubleshooting_opmath` section!
+
+    After visiting the :ref:`Troubleshooting_opmath` section, if you are still stuck then you can:
+
+    - Post on the PennyLane `discussion forum <https://discuss.pennylane.ai>`_.
+
+    - If you suspect that your issue is due to a bug in PennyLane itself, please open a
+      `bug report <https://github.com/PennyLaneAI/pennylane/issues/new?labels=bug+%3Abug%3A&template=bug_report.yml&title=[BUG]>`_
+      on the PennyLane GitHub page.
+
+Summary of the update
+---------------------
+
+.. rst-class:: admonition tip
+
+    The opt-in feature ``qml.operation.enable_new_opmath()`` is now the default. Ideally, your code should not break.
+    If it still does, it likely only requires some minor changes. For that, see the :ref:`Troubleshooting_opmath` section.
+    You can still opt-out and run legacy code via ``qml.operation.disable_new_opmath()``.
+
+
+* The underlying system for performing arithmetic with operators has been changed. Arithmetic can be carried out using
+  standard Python operations like ``+``, ``*`` and ``@`` or via arithmetic functions located in :mod:`~.op_math`.
+
+* You can now easily access Pauli operators via :class:`~.I`, :class:`~.X`, :class:`~.Y`, and :class:`~.Z`.

```

Comment:
these links also dont work

Reply:
Is it because of how we are aliasing `X = PauliX`? Maybe `:class:` is not the right name here. Also, I wouldn't hold up merging the PR for this.

Reply:
don't work for me either, but agree with Tom that this shouldn't hold it up.

Reply:
@Qottmann it might honestly be a PR preview thing, not sure though. Let's see when this gets merged, and if there's still an issue we can just make a new PR with the fixes. Happy to handle that myself :)

Reply:
That would be awesome, thanks!